### ISSUE TITLE
Matplotlib Backend "PySide" SegFaults

### ISSUE THREAD

**Issue by aherrmann** [NixOS/nixpkgs#12798]
I have noticed, that the pyside backend to matplotlib generates a segmentation fault on the latest checkout from the nixpkgs-unstable channel, this used to work before. With git bisect I was able to reduce the set of possibly guilty commits to the following three.

* 798c5469d94d85947beb7f2ed475aa6c9dc5b91f
* d320d3dcbf0eefab285496481aa32d7926c6f648
* df6a869ea71bc449eb97c8b5940e13f28452b01e

I suspect that it has to do with the update to matplotlib 1.5.0.

---

To reproduce the error save the following as `test.py` make it executable and run as `nix-shell test.py`.

``` python
#!/usr/bin/env nix-shell
#!nix-shell -i python -p python
#!nix-shell -p pyside pythonPackages.matplotlib

import matplotlib as mpl
mpl.rcParams['backend.qt4'] = 'PySide'
mpl.use('Qt4Agg')
import matplotlib.pyplot as plt

plt.plot(range(1,5), range(3,7))
plt.show()
```

The script produces just the following output.

```
Segmentation fault
```

---

It works just fine with PyQt4, and PyQt5.

``` python
#!/usr/bin/env nix-shell
#!nix-shell -i python -p python
#!nix-shell -p pythonPackages.pyqt4 pythonPackages.matplotlib

import matplotlib as mpl
mpl.rcParams['backend.qt4'] = 'PyQt4'
mpl.use('Qt4Agg')
import matplotlib.pyplot as plt

plt.plot(range(1,5), range(3,7))
plt.show()
```

``` python
#!/usr/bin/env nix-shell
#!nix-shell -i python -p python
#!nix-shell -p pythonPackages.pyqt5 pythonPackages.matplotlib

import matplotlib as mpl
mpl.use('Qt5Agg')
import matplotlib.pyplot as plt

plt.plot(range(1,5), range(3,7))
plt.show()
```

**Comment by aherrmann** [comment]
@FRidh I just saw that you updated matplotlib (#13078). I tried it and the issue seems to be resolved on the current master branch (b3f764428f87ad2b4b3fb32de350a404272ce40e). Thanks!

**Comment by FRidh** [comment]
Great, thanks for reporting it was fixed.
